open_academy/views/open_academy_course_views.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
     <!-- Action of the view configuration -->
-    <record model="ir.actions.act_window" id="course_action_list">
+    <record model="ir.actions.act_window" id="open_academy_course_action_list">
```

Comment:
First ID then model. See the guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action
```suggestion
        <record id="open_academy_course_action_list" model="ir.actions.act_window">
```

Please apply this suggestion in all records.